example_yamls/example_tunet.yaml

```diff
@@ -0,0 +1,39 @@
+# Example for parameters to excecute
+
+# I/O
+data_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/rec20190524_085542_clay_testZMQ_8bit/20190524_085542_clay_testZMQ_
+data_tiled_api_key: <secret>
+mask_tiled_uri: https://tiled-seg.als.lbl.gov/api/v1/metadata/reconstruction/seg-partial-rec20190524_085542_clay_testZMQ_8bit/seg-partial-20190524_085542_clay_testZMQ_
+mask_tiled_api_key: <secret>
+seg_tiled_uri: http://0.0.0.0:8888
+seg_tiled_api_key: <secret>
+
+mask_idx: [10, 201, 222, 493]
```

Comment:
As mentioned above for Dylan's comment, the `mask_idx` and `shift` will be taken out according to an upcoming PR from the seg app for mask writing. So the app will not send the mask indexes as a parameter, and if I recall correctly, it will be a part as the metadata from tiled.

Please correct me if I am wrong @Wiebke .